Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -601,7 +601,7 @@ def genS(fragment,howMuch):
 steps['AMPT_PPb_5020GeV_MinimumBias']=merge([{'-n':10},step1PPbDefaults,genS('AMPT_PPb_5020GeV_MinimumBias_cfi',Kby(9,100))])
 
 ## pPb Run2
-step1PPbDefaultsUp15={'--beamspot':'Run2PPbBoost','--conditions':'auto:run2_mc_pa','--eventcontent':'RAWSIM', '--era':'Run2_2016_pA'}
+step1PPbDefaultsUp15={'--beamspot':'Realistic50ns13TeVCollision','--conditions':'auto:run2_mc_pa','--eventcontent':'RAWSIM', '--era':'Run2_2016_pA'}
```

Comment:
@mandrenguyen @jrcastle in light of what is requested [https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2775.html](here), when changing the RECO BeamSpot to `BeamSpotObjects_Realistic8TeVPACollision2016_v1` in the `auto:run2_mc_pa` Global Tag, shouldn't be the default vertex smearing changed as well?
   * would you mind if I update the autoCond as part of this PR adding commits to yours?
   * can you clarify at that point which is the preferred `--beamspot` option you'd like to have in relvals? is it `RealisticPPbBoost8TeV2016Collision` or `RealisticPbPBoost8TeV2016Collision` ?
Thanks,

M. 

@franzoni @arunhep